R/colour-text.r

```diff
@@ -5,9 +5,13 @@ testthat_colours <- list(
   error = crayon::red
 )
 
-colourise <- function(text, as = c("passed", "skipped", "error")) {
+#' @export
+colourise <- function(text, as = c("passed", "skipped", "error"), fg, bg) {
```

Comment:
I think this is now too complicated - it would be better to have two functions. One function should be exported and use the old interface and wrap the equivalent crayon function with a deprecation message. The other function should be internal and be the same as the current `colourise()` but with a different name.